### DISCUSSION TITLE
New Ideas for Docker for Unraid.

### DISCUSSION THREAD

**Discussion by P-C-R** [accetto/ubuntu-vnc-xfce-g3#19] [Ideas]
I would love to see a Container for

* Freecad Linkstage3 Fork
* PrusaSlicer
;)

**Comment by accetto** [comment]
Hello @P-C-R ,

thank you for your feedback. I have to apologize, that I haven't noticed it for weeks, unfortunatelly.

Currently I'm working on new releases, but I'll will definitely check your proposals.

Regards,
accetto
